src/modules/mount/main.py

```diff
@@ -112,25 +110,14 @@ def mount_partition(root_mount_point, partition, partitions):
         if "luksMapperName" in partition:
             device = os.path.join("/dev/mapper", partition["luksMapperName"])
 
-        if libcalamares.utils.mount(device,
+        # Mount the subvolumes
+        for s in btrfs_subvolumes:
+            mount_option = f"subvol={s['subvolume']}"
+            if libcalamares.utils.mount(device,
                                     mount_point,
                                     fstype,
-                                    ",".join(["subvol=@", partition.get("options", "")])) != 0:
-            libcalamares.utils.warning("Cannot mount {}".format(device))
-
-        if not has_home_mount_point:
-            if libcalamares.utils.mount(device,
-                                        root_mount_point + "/home",
-                                        fstype,
-                                        ",".join(["subvol=@home", partition.get("options", "")])) != 0:
-                libcalamares.utils.warning("Cannot mount {}".format(device))
-        
-        if needs_swap_subvolume:
-            if libcalamares.utils.mount(device,
-                                        root_mount_point + "/swap",
-                                        fstype,
-                                        ",".join(["subvol=@swap", partition.get("options", "")])) != 0:
-                libcalamares.utils.warning("Cannot mount {}".format(device))
+                                    ",".join([mount_option, partition.get("options", "")])) != 0:
+                libcalamares.utils.warning(f"Cannot mount {device}")
```

Comment:
Using f-strings makes this a Calamares 3.3 PR, because f-strings are a newer Python feature than the Python supported by Calamares.

Reply:
Good to know. When is calamares 3.3 release planned? I can adjust it to another style if 3.3 is far away, but it kinda feels silly to change it to less efficient and older form.

Reply:
F-strings removed and replaced with format() method